Firestore/core/src/firebase/firestore/util/status.cc

```diff
@@ -116,11 +116,6 @@ void Status::IgnoreError() const {
   // no-op
 }
 
-std::ostream& operator<<(std::ostream& os, const Status& x) {
```

Comment:
We don't use these stream insertion operators so this seems like a reasonable work around. Please also remove the declaration in status.h.